TASK-5/TASK-5/Menu.cs

```diff
@@ -0,0 +1,172 @@
+
+            switch (menuItem)
+            {
+                case 1:
+                    DisplayEveryClient(users);
+                    break;
+                case 2:
+                    DisplayClientPurchases(users);
+                    break;
+                case 3:
+                    GenerateUserFromConsole(users);
+                    break;
+                case 4:
+                    AddItemToBag(users);
+                    break;
+                case 5:
+                    RemoveItemFromBag(users);
+                    break;
+                default:
+                    exit = false;
+                    break;
+            }
+        }
+    }
+
+    public static List<User> GenerateUsers(int count)
+    {
+        var users = new List<User>();
+        for (int i = 0; i < count; i++)
+        {
+            users.Add(UserGenerator.CreateUser());
+        }
+        
+        return users;
+    }
+
+    private static void DisplayEveryClient(List<User> usersContainer)
+    {
+        var counter = 0;
+        Console.WriteLine();
+        foreach (var user in usersContainer)
+        {
+            counter++;
+            Console.WriteLine($"{counter}. {user.FullName} || Age is {user.Age} || ID is {user.PassportId}");
+        }
+    }
+
+    private static void DisplayClientPurchases(List<User> users, int userNumber)
+    {
+        decimal totalPrice = 0;
+        foreach (var items in users[userNumber-1].GroceryBag)
+        {
+            Console.WriteLine($"{items.Barcode} || {items.Category} || {items.Price}$ || {items.ItemName}");
+            totalPrice += items.Price;
+        }
+        Console.WriteLine($"Total: {totalPrice}$");
+    }
+    
+    private static void DisplayClientPurchases(List<User> users)
+    {
+        decimal totalPrice = 0;
+        string input;
+        int itemNumber;
+        do
+        {
+            Console.WriteLine("Please, enter User number (starts from 1): ");
+            input = Console.ReadLine();
+        } while (!int.TryParse(input, out itemNumber));
+        
+        foreach (var items in users[itemNumber-1].GroceryBag)
+        {
+            Console.WriteLine($"{items.Barcode} || {items.Category} || {items.Price}$ || {items.ItemName}");
+            totalPrice += items.Price;
+        }
+        Console.WriteLine($"Total: {totalPrice}$");
+    }
+    
+   private static void GenerateUserFromConsole(List<User> users) 
+    {
+        Console.WriteLine("Enter new user's ID");
+        var userPassportIdInput = Console.ReadLine();
+        Guid passportId;
+        while (string.IsNullOrWhiteSpace(userPassportIdInput) || userPassportIdInput.Length != 32 
+                                                              || !Guid.TryParse(userPassportIdInput, out passportId))
+        {
+            Console.WriteLine("User's passport ID cannot be empty and it's length has to be 32 symbols");
+            userPassportIdInput = Console.ReadLine();
+        }
```

Comment:
Make validation method with regex for passportId(In Validation class, for example)